project/tests/test_pentane_ua.py

```diff
@@ -0,0 +1,19 @@
+import mbuild as mb
+import numpy as np
+import pytest
+
+from project.src.molecules.pentane_ua import PentaneUA
+from project.tests.base_test import BaseTest
+
+
+class TestPentaneUA(BaseTest):
+    """Tests to ensure PentaneUA behaves as expected."""
+
+    def test_creation(self):
+        pent = PentaneUA()
+        assert isinstance(pent, mb.Compound)
+        assert pent.n_particles == 5
+        assert pent.n_bonds == 4
+        assert pent.name == "PentaneUA"
+        assert all([p.name in ["_CH2", "_CH3"] for p in pent.particles()])
+        assert len(pent.labels) == 1
```

Comment:
Check one bond length and one angle as well? Might not be required though.

```python 
np.linalg.norm(pentane.xyz[3]-pentane.xyz[0])*10
```
```python


a = pentane.xyz[0]
b = pentane.xyz[1]
c = pentane.xyz[2]

ba = a - b
bc = c - b

cosine_angle = np.dot(ba, bc) / (np.linalg.norm(ba) * np.linalg.norm(bc))
angle = np.arccos(cosine_angle)

print(np.degrees(angle))
```

Reply:
Angle calculation code taken from here: https://stackoverflow.com/questions/35176451/python-code-to-calculate-angle-between-three-point-using-their-3d-coordinates